compiler/optimizer/SwitchAnalyzer.cpp

```diff
@@ -1257,8 +1255,6 @@ TR::Block *TR::SwitchAnalyzer::addTableBlock(SwitchInfo *dense)
 
    CASECONST_TYPE upperBound = dense->_max - dense->_min;
 
-   TR::SymbolReference *branchTableSymRef = NULL;
-
    int32_t branchTable = 0;
```

Comment:
Is this variable also always zero and unused once the dead code you have written is cleaned up? If so could you tidy this up too?